config/settings/base.py

```diff
@@ -228,3 +229,7 @@
 # Should always be False in production. Can be set to True in local environments
 # to serve static files even when DEBUG is False
 DJANGO_SERVE_STATIC = False
+
+CLIENT_BASE_URL = os.getenv("CLIENT_BASE_URL")
+CLIENT_KEY = os.getenv("CLIENT_KEY")
+CLIENT_VERIFY_CERTIFICATES = True
```

Comment:
We should still be able to overwrite this is there are any issues but just have the default be `True`:

```suggestion
CLIENT_VERIFY_CERTIFICATES = strtobool(
    os.getenv("CLIENT_VERIFY_CERTIFICATES", "True")
)
```

Reply:
As with the above variables, would this be better as `ROSETTA_API_VERIFY_CERTIFICATES`?

Reply:
`CLIENT_VERIFY_CERTIFICATES = True` must always be True, specially for production, otherwise the application can be vulnerable to attacks.  I've set it to True so that we don't exclusively need to set that in production. But handle development with a parametrised value.

Reply:
Imagine that for whatever reason the certificates on CIIM/Rosetta expired once the search was in production and they couldn't quickly or easily be fixed.

I would like a way for our production site to not have to verify them so it can continue to work.